### ISSUE TITLE
[WIP] moving to beta release 

### ISSUE THREAD

**Issue by andyxning** [kubernetes-incubator/node-feature-discovery#74]
Before we make node-feature-discovery to a beta release, we need to do things below finished.

- [ ] move import path from `github.com/kubernetes-incubator` to `k8s.io`.
- [ ] make an agreement on the label schema about node-feature-discovery. #71
- [ ] adjust node-feature-discovery to meet the requirements to be a Kubernetes repo described in [kubernetes-template-project](https://github.com/kubernetes/kubernetes-template-project).

**Comment by andyxning** [comment]
@marquiz Any docs or record video about the discussion.

**Comment by marquiz** [comment]
Hmm, it seems that the last recording from SIG-node meeting is from April, so I'm afraid the answer is no. But, this was still a preliminary discussion. I think there needs to more evidence on the health of the nfd project and its benefits before the move under sig-node will be considered

**Comment by andyxning** [comment]
> I think there needs to more evidence on the health of the nfd project and its benefits before the move under sig-node will be considered

Agreed. We need first make nfd more common, imo, the support for the hardware detect is somewhat too detail. and collect more user cases.

**Comment by marquiz** [comment]
> As for the detailed collect, it do not mean that it is not necessary, but maybe some common useful ones are more demanded.

Hmm, I don't know if I quite understand what you mean here. Do you have some specific features in mind?
